frontend/test/metabase-smoketest/admin_setup.cy.spec.js

```diff
@@ -552,11 +586,36 @@ describe("smoketest > admin_setup", () => {
   describe("permission changes reflected", () => {
     beforeEach(signInAsNormalUser);
 
-    it("should check current permissions as use", () => {});
+    it("should check current permissions as users", () => {
+      // Access to all tables as user
+      cy.visit("/");
+
+      cy.contains("A look at your People table");
+      cy.contains("A look at your Test Table table");
+      cy.findByText("A look at your Review table").should("not.exist");
+
+      // Access to SQl queries as user
+
+      cy.findByText("Ask a question").click();
+      cy.findByText("Native query");
+
+      // Cannot see Test Table or People table as no data user
+
+      signOut();
+      signIn("nocollection");
+      cy.visit("/");
+
+      cy.contains("A look at your People table").should("not.exist");
```

Comment:
Before testing for something that `should("not.exist")` we should do another test to ensure the page loaded. So in this case maybe we wait for "Try these x-rays" to show up.